gui/default/syncthing/core/syncthingController.js

```diff
@@ -1733,15 +1733,17 @@ angular.module('syncthing.core')
 
         $scope.setAllDevicesPause = function (pause) {
             for (var id in $scope.devices) {
-                $scope.devices[id].paused = pause;
+                if ($scope.devices[id].deviceID !== $scope.myID) {
+                  $scope.devices[id].paused = pause;
+                }
```

Comment:
Since this was required in multiple places and we're liable to forget it in the future, maybe wrap the condition into an iterator like `$scope.forAllRemoteDevices(fn)` or so?

Reply:
Seems sensible - what's your "status" on this, @tomasz1986?

Reply:
I have tried a different approach on this. What do you think about the current iteration?